UnityProject/Assets/Scripts/Tilemaps/Behaviours/Layers/MetaTileMap.cs

```diff
@@ -606,6 +606,16 @@ private bool IsPassableAtOrthogonalTileV2(Vector3Int origin, Vector3Int to, Coll
 
 				if (tile.IsPassable(colliderType, origin, this) == false)
 				{
+					foreach (var interaction in tile.TileInteractions)
+					{
+						if (interaction is IBumpableObject bump)
+						{
+							if (Bumps.Contains(bump) == false)
+							{
+								Bumps.Add(bump);
```

Comment:
I did this same exact thing yesterday, why does it work in this context now?

Reply:
it seems to work fine?

Reply:
if you debug it and name all tiles that got interacted with, does it list all tiles correctly or does it just completely miss all tiles unless you're looking upwards?